website/docs/components/alert/partials/content/content.md

```diff
@@ -0,0 +1,151 @@
+## General recommendations
```

Comment:
@jorytindall similar for the code showcase, this content (or part of it) should also be replicated/adapted for the `Toast` component, that inherits all the behaviours/functionalities/styles from the `Alert`.

Reply:
Ok, I've made several updates to the content, broken down the recommendations a bit. Let me know what you think, if it works I can translate/copy relevant content to the Toast.

Reply:
It works™